### PR TITLE
Remove uneeded syntax

### PR DESCRIPTION
**Pull request by alexprut**
**Nitpicky Small Changes**
* Remove uneeded convertion type
* Remove redundant type in composite literal
* Omit explicit type where implicit
* Remove unused redundant parenthesis

### CODE REVIEW

**Comment by alexprut** [comment]
Given that this PR got out of sync already once, the benefit of keeping it in sync is less that the actual gains of this small syntactic sugar simplification changes. 

Would you mind reviewing @fjl @karalabe @holiman 🙏 .
Otherwise feel free to close this PR. The build is also misbehaving, probably some flaky tests?

File: metrics/cpu_syscall.go [hunk 1]
```diff
@@ -31,5 +31,5 @@ func getProcessCPUTime() int64 {
 		log.Warn("Failed to retrieve CPU time", "err", err)
 		return 0
 	}
-	return int64(usage.Utime.Sec+usage.Stime.Sec)*100 + int64(usage.Utime.Usec+usage.Stime.Usec)/10000 //nolint:unconvert
+	return usage.Utime.Sec + usage.Stime.Sec*100 + int64(usage.Utime.Usec+usage.Stime.Usec)/10000 //nolint:unconvert
```
Comment (karalabe) [inline metrics/cpu_syscall.go:34]: You messed up the math, parenthesis was needed

**Review by holiman — APPROVED** [review]
LGTM

File: metrics/cpu_syscall.go (hunk 1 above)
Comment (alexprut) [reply metrics/cpu_syscall.go]: Ouch, Fixed! 🙏

**Review by karalabe — APPROVED** [review]
LGTM thx!

**Pull request merged**